truffle/src/main/java/org/jruby/truffle/core/string/StringNodes.java

```diff
@@ -23,7 +23,7 @@
  * Copyright (C) 2006 Ola Bini <ola@ologix.com>
  * Copyright (C) 2007 Nick Sieger <nicksieger@gmail.com>
  *
- * Some of the code in this class is transposed from org.jruby.util.ByteList,
+ * Some of the code in this class is transposed from org.jruby.util.ByteList and org.jruby.RubyString,
```

Comment:
This should go away. We note that we take from `org.jruby.RubyString` in the copyright notice above this one.